composer.json

```diff
@@ -21,7 +21,7 @@
     ],
     "homepage": "https://docs.sonata-project.org/projects/SonataPageBundle",
     "require": {
-        "php": "^8.1",
+        "php": "^8.2",
```

Comment:
we are planning to drop support for 8.1 anyway when adding Symfony 8 support. So this change is fine :+1: